### PR TITLE
feat: PLP product multi-dimensional price range

### PR DESCRIPTION
**Pull request by kpawelczak**
closes: [CXSPA-8116](https://jira.tools.sap/browse/CXSPA-8116)

### CODE REVIEW

File: feature-libs/product-multi-dimensional/base/occ/product-multi-dimensional-base-occ.module.ts [hunk 1]
```diff
@@ -4,13 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { provideDefaultConfig } from '@spartacus/core';
 import { defaultOccProductMultidimensionalBaseConfig } from './config/default-occ-product-multi-dimensional-base-config';
```
Comment (Matejk00) [inline feature-libs/product-multi-dimensional/base/occ/product-multi-dimensional-base-occ.module.ts]: nitpick: should we keep the convention and name this variable as: `defaultOccProductMultiDimensionalBaseConfig`?

File: feature-libs/product-multi-dimensional/base/root/components/price-range/product-multi-dimensional-price-range.component.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  getProductPrice(product: Product) {
+    const priceRange = product.priceRange;
+    const isMultiDimensionalAndHasPriceRange =
+      product.multidimensional && Object.values(priceRange).length;
+
+    if (isMultiDimensionalAndHasPriceRange) {
+      const maxPrice = priceRange.maxPrice.formattedValue;
+      const minPrice = priceRange.minPrice.formattedValue;
+      return minPrice + ' - ' + maxPrice;
```
Comment (Matejk00) [inline feature-libs/product-multi-dimensional/base/root/components/price-range/product-multi-dimensional-price-range.component.ts]: I think more frequently we are using backticks method to concat variables and strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

**Review by Matejk00 — CHANGES_REQUESTED** [review]

File: feature-libs/product-multi-dimensional/base/root/components/price-range/product-multi-dimensional-price-range.component.ts (hunk 2 above)
Comment (kpawelczak) [reply feature-libs/product-multi-dimensional/base/root/components/price-range/product-multi-dimensional-price-range.component.ts]: sure, I've changed it

File: feature-libs/product-multi-dimensional/base/occ/product-multi-dimensional-base-occ.module.ts (hunk 1 above)
Comment (kpawelczak) [reply feature-libs/product-multi-dimensional/base/occ/product-multi-dimensional-base-occ.module.ts]: good catch, thanks, fixed

**Review by Matejk00 — APPROVED** [review]

**Pull request merged**